core/src/main/resources/hudson/node_monitors/DiskSpaceMonitor/column.jelly

```diff
@@ -29,7 +29,19 @@ THE SOFTWARE.
       <td align="right" data="-1">N/A</td>
     </j:when>
     <j:otherwise>
-      <td align="right" data="${data.freeSize}"><j:out value="${data.toHtml()}"/></td>
+      <td align="right" data="${data.freeSize}" data-total="${data.totalSize}">
+        <j:choose>
+          <j:when test="${data.triggered}">
+            <span class="error" tooltip="${data.toString()}">${data.toHtml()}</span>
```

Comment:
I thought that I already added a comment with this picture, but now I can't find it.  Apologies if this is a duplicate.

The formatting is not applied to the free disk space warning in my interactive use.  It looks like this when a free disc space problem has taken the agent offline:

![disk-space-monitoring-improvements](https://github.com/jenkinsci/jenkins/assets/156685/67d875bb-4f3a-4a21-85be-962dde3b4f70)

Reply:
That must be it.  Will try later today.

Reply:
Confirmed that it was my mistake.  Nothing wrong with pull request.  Resolving.